templates-and-layout/locked-down-template-inline-bordered.html

```diff
@@ -50,17 +50,32 @@
             // https://www.tiny.cloud/docs/configure/editor-appearance/#fixed_toolbar_container
             fixed_toolbar_container: '#toolbar',
 
+            // Make the toolbar always visible (requires TinyMCE 5.5)
+            // https://www.tiny.cloud/docs/configure/editor-appearance/#toolbar_persist
+            toolbar_persist: true,
+
             setup: (editor) => {
-                // Apply a custom class to the wrapper element when the editor gets focus
-                // https://www.tiny.cloud/docs/advanced/events/
+                // Disable the UI when the editor has loaded
+                // TODO: Use init or PreInit?
+                // TODO: Add documentation for the ui.disable API (Requires TinyMCE 5.6)
+                //https://www.tiny.cloud/docs/advanced/events/
+                editor.on('init', () => {
+
+                    editor.ui.disable();
```

Comment:
When you'll be able to do this on depends on how we solve [TINY-6624](https://ephocks.atlassian.net/browse/TINY-6624). As you have discovered, all components need to have rendered before running `editor.ui.disable()` makes sense in in this use- case.

Reply:
As mentioned by Lee, listen to the 'SkinLoaded' event

Reply:
Now using the SkinLoaded event!